### ISSUE TITLE
[A11y] Search Results - Grid - High Contrast

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1984]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

For some reason the "Image" icon disappears in high contrast mode:

![image](https://user-images.githubusercontent.com/2157974/130692612-26e5f1e1-2291-41ce-ae3c-b6853e11d668.png)

The icon may not be critical, but the presence of a "blank box" is still likely a failure simply because it isn't clear what's supposed to be there.

### Suggested Fix

Font icons and some CSS "decorations" seem to have a lot of odd issues with high-contrast mode. It's probably too late for this, but it may be worth investigating using SVGs or something.  Otherwise, I'm not sure - pick another icon and see if it works in high-contrast mode?

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

Turn on windows high contrast mode and make sure the icons look good.

**Comment by azle** [comment]
Updated 12/30/2024

@jsimic I am not able to recreate the issue on MAC across all browsers. Can someone with a PC check Edge?
